settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "app_settings": "",
```

Comment:
what are possible values of "app_settings"? I feel like it should be removed if you are not planning to use it.

Reply:
I am planning to use it, but for now I don't have a particular setting for the app. I am leaving it there for future use.